helm-chart-sources/appscope/README.md

```diff
@@ -0,0 +1,75 @@
+
+ `helm install appscope cribl/appscope --set appscope.destination.cribl="tcp://in.main-default-<your-organization>.cribl.cloud:10091"`
+
+By default, Cribl.Cloud-managed instances of Cribl Stream have port `10091` configured to use TCP, and a built-in AppScope Source to receive data from AppScope. 
+
+To send metrics & events to a local [Stream](https://cribl.io/stream/) instance, deploy with chart with the `cribl` config value defined as: `cribl-internal:10090`:
+
+ `helm install appscope cribl/appscope --set appscope.destination.cribl="tcp://cribl-internal:10090"`
+
+## Basic Configuration for Cribl Edge 
+
+* To install the chart with the release name "appscope" and send metrics & events to [Cribl Stream](https://cribl.io/stream/) defined as: `cribl-internal:10090`:
+
+ `helm install appscope cribl/appscope --set appscope.destination.cribl="tcp://cribl-internal:10090"`
+
+## Basic Configuration for Statsd Prometheus Exporter
+
+To send `statsd` metrics to a local Statsd Exporter deployment, use the following configuration:
+
+```yaml
+# Appscope configuration 
+appscope:
+  destination: 
+    metrics: <prometheus-statsd-exporter-name>.<namespace>:9125
+  format: "statsd"
+```
+
+# Support/Feedback
+
+If you use this helm chart, we'd love to hear any feedback you might have on this chart. Join us on our [Slack Community](https://cribl.io/community) and navigate to the `#appscope` or `containers` channel.
+
+# More Info
+
+For additional documentation about the values used in this chart, see the [Cribl Docs](https://appscope.dev/docs/cli-reference/#k8s) page.
+
+# Values to Override
+
+This section covers the most likely values to override. To see the full scope of values available, run `helm show values cribl/appscope`.
+
+| Key                                                                            | Default Value     | Description                                        |
+|--------------------------------------------------------------------------------|-------------------|----------------------------------------------------|
+| image.repository                                                               | `cribl/scope`     | Docker image repository to pull images             |
+| image.pullPolicy                                                               | `Always`          | When will the Node pull the image                  |
+| image.tag                                                                      | `1.3.2`           | The Version of Appscope to deploy                  |
+| imagePullSecrets                                                               | `[]`              | Credentials to pull container images               |
+| appscope.destination.cribl                                                     |                   | Cribl Stream destination for metrics & events      |
```

Comment:
nit: put see `values.yaml` in the default value column for complex configurations, otherwise someone might assume we accidentally left this blank.

Reply:
The `appscope.destination.cribl`, `appscope.destination.metrics`, `appscope.destination.events` are empty here by intention.
See  https://github.com/criblio/helm-charts/blob/e7e730ba1e0c9f15760d94aabc168965e5f199c0/helm-chart-sources/appscope/templates/configmap.yaml#L8